src/app/util/util.ts

```diff
@@ -40,4 +50,30 @@ export class Util {
     }
   }
 
+  /**
+   * Creates notification instance according to its type
+   */
+  public static createNotificationInstance<T extends AbstractNotification>(notification: any): T {
+    switch (notification.type) {
+      case NotificationType.ACCEPTANCE_TO_COMMUNITY:
+        return Object.assign(new AcceptanceToCommunityNotification, notification);
```

Comment:
Aren't the brackets necessary when calling a constructor?

Reply:
thank you. done